components/widget/match/summary/ffa/widget_match_summary_ffa_tab.lua

```diff
@@ -24,7 +25,7 @@ function MatchSummaryFfaTab:render()
 			['data-js-battle-royale'] = 'panel-content',
 			id = self.props.matchId .. 'panel' .. self.props.idx,
 		},
-		children = self.props.children,
+		children = WidgetUtil.collect(unpack(self.props.children or {})),
```

Comment:
Better to change the customs to use WidgetUtil.collect instead

Reply:
okay